### ISSUE TITLE
NTSCQT Rendering takes too long

### ISSUE THREAD

**Issue by MasterNama17083** [JargeZ/vhs#24]
Hello. I'm rendering video frames I know what's happening there, probably I have 8GB RAM on my laptop.
![image](https://user-images.githubusercontent.com/80534869/147244627-61b0f219-5b9a-47a5-aac8-36abc409e44f.png)

It takes **2,012.8 MB** may slow down CPU Usage.
![image](https://user-images.githubusercontent.com/80534869/147244787-b8e08aec-6390-43e0-a961-c1de55a76521.png)

What I do now takes spend rendering more minutes?

**Comment by Supergub5871** [comment]
do life

**Comment by marine025** [comment]
its too excessive is the main problem with this app is the best but i prefer other worse emulators

**Comment by BugBerry202** [comment]
no i think it's because it's only using one of your cpu cores, we need multi-core rendering

**Comment by marine025** [comment]
how i could do that??!! say to my computer (katana msi i7) use all your energy in this program, seems like is a problem of that, because isnt make sense the amount of time, any oher program of my computer is faster doesnt matter the amount of work i do

**Comment by BugBerry202** [comment]
ntscqt doesn't have all of the power of your cpu, if you notice here it's only using one of my 12 threads, making it very slow unfortunately.
![image](https://github.com/JargeZ/ntscqt/assets/68083196/9c8f3340-c306-4472-b0b2-00c075d8eeea)

**Comment by MasterNama17083** [comment]
Ah, it's been three years ago. I remember exporting video with NTSCQT that took 1-2 hours to render but don't worry it's just all fine.

**Comment by marine025** [comment]
no, the issue persist, maybe is me who use to render videos of 1 hour or more but its takes a whole day or more even in 720p

**Comment by marine025** [comment]
can x2 output by nearest neighbor increase the speed? can increase the speed if i execute as admin and i select maximun priority in task manager? what's the way to derivate more power to the program
